### ISSUE TITLE
Header, Link, and Labeling Styles Confusing

### ISSUE THREAD

**Issue by ntallman** [psu-libraries/cho#801]
It's hard to distinguish between headers, metadata field labels, unclicked links, and clicked links. They are all black or shades of gray.

**Comment by awead** [comment]
Blocked by #787

**Comment by mtribone** [comment]
I believe that this is no longer being blocked https://github.com/psu-libraries/cho/pull/787
